src/Components/ADT3DViewer/ADT3DViewer.tsx

```diff
@@ -108,12 +108,16 @@ const ADT3DViewerBase: React.FC<IADT3DViewerProps> = ({
     const prevLayerCount = useRef<number>(-1); // track the count of layers so we know to refresh the selections if the count changes
     const hasUserChangedLayers = useRef<boolean>(false); // need to know once a user makes a selection so we stop auto selecting items
     const prevSceneId = useRef<string>(sceneId); // need to know if user swaps scenes (using scene dropdown) since we won't get remounted
+    const initialDeeplinkLayers = useRef<string[]>(
```

Comment:
nit: for refs we used to use Ref at the end of the name of the variable to differentiate from other state variables but it seems all the useRefs are not using that convention, just dropping here for the future.